subtests/docker_cli/run_sigproxy/run_sigproxy.py

```diff
@@ -98,7 +112,7 @@ def initialize(self):
             self._init_container_attached(name)
         else:
             self._init_container_normal(name)
-        time.sleep(self.config['wait_start'])
+        self._wait_for_ready()
```

Comment:
I like that (I didn't know about wait_for_output), thanks! But can we compromise? Keep _wait_for_ready() because it keeps initialize() smaller and more readable? Even though it's a single-use method, I'm a fan of small functions for compartmentalizing head-scratchy stuff.